R/read_csv.R

```diff
@@ -430,16 +430,22 @@ read_csv_metadata <- function(csv_file) {
         inv_metric_split <- strsplit(gsub("# ", "", line), ",")
         if ((length(inv_metric_split) == 0) ||
             ((length(inv_metric_split) == 1) && identical(inv_metric_split[[1]], character(0))) ||
-            grepl("[a-zA-z]", line, perl = TRUE) ||
             inv_metric_split == "#") {
           parsing_done <- TRUE
           parse_key_val <- TRUE
           break;
         }
+        suppressWarnings(
+          numeric_inv_metric_split <- tryCatch(rapply(inv_metric_split, as.numeric), error = function() {
+            parsing_done <- TRUE
+            parse_key_val <- TRUE
+            break;
```

Comment:
I think the use of `break` here is what's causing R cmd check to fail. The tests pass because this code actually runs fine but the CRAN checks don't like it: 

```
Warning: Found the following significant warnings:
  Note: break used in wrong context: no loop is visible 
```

Reply:
Ah yeah, its a copy paste error.